apps/api/tests/test_profiles_api.py

```diff
@@ -0,0 +1,83 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from apps.klasses.models import Klass
+from apps.profiles.models import Instructor
+
+User = get_user_model()
+
+
+class ProfilesIntegrationTests(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create_user(username='user', password='pass')
+        self.instructor = Instructor.objects.create(university_name='Test')
+        self.user.instructor = self.instructor
+        self.user.save()
+        self.klass = Klass.objects.create(instructor=self.instructor, course_number="1")
+
+    def test_get_users_works(self):
```

Comment:
we should get out of the habit of saying "works" -- `test_get_users_returns_proper_user_data`

should maybe assert the length of the thing returned is valid